src/handlers/major_change.rs

```diff
@@ -85,6 +99,39 @@ pub(super) async fn handle_input(
             "This proposal has been accepted: [#{}]({}).",
             event.issue.number, event.issue.html_url,
         ),
+        Invocation::Rename { prev_issue } => {
+            let issue = &event.issue;
+
+            let prev_topic = zulip_topic_from_issue(&prev_issue);
+            let partial_issue: PartialIssue = issue.into();
+            let new_topic = zulip_topic_from_issue(&partial_issue);
+
+            let zulip_send_req = crate::zulip::MessageApiRequest {
+                recipient: crate::zulip::Recipient::Stream {
+                    id: config.zulip_stream,
+                    topic: &prev_topic,
+                },
+                content: "The associated GitHub issue has been renamed. Renaming this Zulip topic.",
+            };
+            let zulip_send_res = zulip_send_req
+                .send(&ctx.github.raw())
+                .await
+                .context("zulip post failed")?;
```

Comment:
Why are we sending a separate message for this? Can we not just rename the topic? It seems like either way I would want to post the message *after* we attempt to rename the topic.

Reply:
I think we discussed this a few months ago, but I think the way you rename a topic in Zulip is to send an UpdateMessage request, but we need a message in the topic to update it, so it sends a new message to the old topic and then updates the new message to use the new topic.

Reply:
Ok. Seems fine.